src/ziggurat/mapper.clj

```diff
@@ -14,6 +13,10 @@
     (throw (ex-info "Invalid mapper return code" {:code return-code})))
   (producer/publish-to-channel-instant-queue return-code message-payload))
 
+(defn- report-errors [throwable message]
```

Comment:
This private function need not belong here. Especially, in the case when another namespace needs to report errors to both sentry and newrelic.